cca_utils/ldap3_utils.py

```diff
@@ -280,27 +280,21 @@ def ldap_change_password(username, raw_password):
 def ldap_add_assurance(username, assurance):
     dn = "uid={username},{ou}".format(username=username, ou=settings.LDAP_PEOPLE_OU)
     mod_attrs = {}
-    mod_attrs['userPassword'] = [MODIFY_ADD, assurance]
+    mod_attrs = {'eduPersonAssurance': [MODIFY_ADD, assurance]}
```

Comment:
you've changed the key here, that's on purpose, right?

Reply:
yea it was a mistake the first time around. We do not want to change the userPassword.